### DISCUSSION TITLE
How to handle FilterableField when the returned value is an enum that has a value that would never be returned in reality

### DISCUSSION THREAD

**Discussion by zilvis97** [TriPSs/nestjs-query#329] [General]
I have a class `DatabaseAccess`
```
@ObjectType()
class DatabaseAccess {
  @FilterableField(() => ExtendedDbAccessStatus, {
    name: 'status',
    allowedComparisons: ['eq', 'in'],
    overrideFilterTypeNamePrefix: 'AccessStatus',
  })
  status: DbAccessStatus;
}
```
value of  status is `DbAccessStatus` which is an enum that looks something like this:
`enum DbAccessStatus {
  Invited = 'invited',
  Active = 'active',
  Deactivated = 'deactivated' }`

I want to be able to filter this field by `ExtendedDbAccessStatus` that would be an  extension of the first enum and look something like this: 
`enum ExtendedDbAccessStatus { ...DbAccessStatus, SomeOtherStatus = 'status' }`

when I generate the schema it will be implied that `status` can be of value `SomeOtherStatus`, but in my case, status would never be this value, in the db it will appear as one of the `DbAccessStatus` values. I simply want to have this value to be able to filter other statuses on the frontend. What would be the best way to do it?

**Comment by TriPSs** [comment]
I tend to create a separate enum with all the values you want available there.
